Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Tasks](https://hosted.weblate.org/projects/kimai/tasks/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Actions (Dropdowns and Toolbars)](https://hosted.weblate.org/projects/kimai/actions/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kimai/messages/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/kimai/kimai?pullRequest=4876) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/kimai/kimai?pullRequest=4876) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/kimai/kimai?pullRequest=4876) it.</sub>

weblate committed:
Translated using Weblate (Persian)

Currently translated at 79.8% (348 of 436 strings)

Translated using Weblate (Faroese)

Currently translated at 23.0% (3 of 13 strings)

Translated using Weblate (Faroese)

Currently translated at 55.9% (244 of 436 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (436 of 436 strings)

Translated using Weblate (Korean)

Currently translated at 69.4% (303 of 436 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 78.8% (344 of 436 strings)

Translated using Weblate (Hungarian)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Korean)

Currently translated at 67.6% (295 of 436 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (436 of 436 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (436 of 436 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (8 of 8 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (436 of 436 strings)

Co-authored-by: Chan Young Park <designbycy@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Oh Wow! <kasra.hashemi1999@gmail.com>
Co-authored-by: René Bischoff <rene.bischoff@gmail.com>
Co-authored-by: Wei-Cheng Yeh (IID) <iid@ccns.ncku.edu.tw>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Co-authored-by: wkwkjddndjdusiq <wkwkjddndjdusiq@onionmail.com>
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/es/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/hu/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/fo/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/es/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fa/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fo/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ko/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/teams/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/wizard/zh_Hant/
Translation: Kimai - Time tracking/Emails
Translation: Kimai - Time tracking/Invoice-Calculator
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Teams
Translation: Kimai - Time tracking/Wizard
